docs/docs/sources/sms-twilio.md

```diff
@@ -0,0 +1,60 @@
+---
+title: SMS with Twilio
+sidebar_label: SMS - Twilio
+---
+
+The Twilio sms source provides a channel for sending and receiving SMS using the [Twilio API](https://www.twilio.com/).
+
+This document assumes that you have already created a Twilio account.
+
+## Configuration
+ 
+First you must create a [Twilio auth token](https://support.twilio.com/hc/en-us/articles/223136027-Auth-Tokens-and-How-to-Change-Them),
+which you then add to `infrastructure/airy.conf` like so:
+
+```
+TWILIO_AUTH_TOKEN=<YOUR AUTH TOKEN>
+```
+
+You also have to add your account sid:
+
+```
+TWILIO_SID=<YOUR ACCOUNT SID>
+```
```

Comment:
I suggested a way to rewrite this so it's a little lighter, I guess we could reuse the markdown 

As I understand it, docusaurus supports it via https://mdxjs.com/getting-started#documents